ls1intum/Artemis
# Feature/compass/impact

### Checklist

### Motivation and Context
Getting too specific feedback from tutors makes automatic suggestions not acceptable for other submissions. Therefore we are giving information about how their feedback can affect other submissions and hoping that tutors will give more general feedback to elements and write specific ones to the feedback below the editor.

### Steps for Testing
1. Log in to Artemis
2. Navigate to Course Management
3. Create an exercise with automatic assessment suggestions enabled and with a due date that will be after you submit all your submissions
4. Submit a submission
5. Log in as a student and submit a similar submission
6. Repeat step 5 if you want more submissions
7. Log in as instructor or tutor (Someone who can assess the submissions) after due date
8. Assess one submission and check if Artemis tells you the number of similar submissions as in the screenshot

### Screenshots
![image](https://user-images.githubusercontent.com/16825065/124084138-f81ebb80-da4e-11eb-97ff-1dcd16697c17.png)
